### PR TITLE
TLS enabled zookeeper

### PR DESCRIPTION
**Pull request by Haarolean**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Replaced the zookeeper client for a few reasons:
1) The previous client we've used does not support TLS as it seems
2) The author/maintainer or the client we've used has been inactive on github since 2018-ish with all the emerging issues.
As a result I had to add the maven central repo to fetch the official apache zookeeper client dependency since it's not being present in confluent repo.

**Is there anything you'd like reviewers to focus on?**
I've provided an example docker-compose file which supports TLS enabled zookeeper. Tell me if I need to commit the truststore/keystore files as well.

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [ ] Unit checks
- [x] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Review by IldarAlmakaev — COMMENTED** [review]
Overall, looks good to me.
Good job, @Haarolean !

**Review by IldarAlmakaev — APPROVED** [review]

**Review by MarselAhmetov — APPROVED** [review]

File: docker/kafka-ui-zookeeper-ssl.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      - kafka-connect0
+    environment:
+      KAFKA_CLUSTERS_0_NAME: local
+      KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: kafka0:29092
+      KAFKA_CLUSTERS_0_ZOOKEEPER: zookeeper0:2182
+      KAFKA_CLUSTERS_0_JMXPORT: 9997
+      KAFKA_CLUSTERS_0_SCHEMAREGISTRY: http://schemaregistry0:8085
+      KAFKA_CLUSTERS_0_KAFKACONNECT_0_NAME: first
+      KAFKA_CLUSTERS_0_KAFKACONNECT_0_ADDRESS: http://kafka-connect0:8083
+      ZOOKEEPER_CLIENTCNXNSOCKET: org.apache.zookeeper.ClientCnxnSocketNetty
```
Comment (germanosin) [inline docker/kafka-ui-zookeeper-ssl.yml:27]: This might not work for all cases it's better to make Zookeeper TLS config on cluster level

**Review by germanosin — CHANGES_REQUESTED** [review]

File: docker/kafka-ui-zookeeper-ssl.yml (hunk 1 above)
Comment (Haarolean) [reply docker/kafka-ui-zookeeper-ssl.yml]: @germanosin thank you. Done.

File: docker/kafka-ui-zookeeper-ssl.yml [hunk 2]
```diff
@@ -24,12 +24,12 @@ services:
       KAFKA_CLUSTERS_0_SCHEMAREGISTRY: http://schemaregistry0:8085
       KAFKA_CLUSTERS_0_KAFKACONNECT_0_NAME: first
       KAFKA_CLUSTERS_0_KAFKACONNECT_0_ADDRESS: http://kafka-connect0:8083
-      ZOOKEEPER_CLIENTCNXNSOCKET: org.apache.zookeeper.ClientCnxnSocketNetty
-      ZOOKEEPER_CLIENT_SECURE: 'true'
-      ZOOKEEPER_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.keystore.jks
-      ZOOKEEPER_SSL_KEYSTORE_PASSWORD: 12345678
-      ZOOKEEPER_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.truststore.jks
-      ZOOKEEPER_SSL_TRUSTSTORE_PASSWORD: 12345678
+      ZOOKEEPER_CLUSTERS_0_CLIENTCNXNSOCKET: org.apache.zookeeper.ClientCnxnSocketNetty
```
Comment (germanosin) [inline docker/kafka-ui-zookeeper-ssl.yml:27]: Could we use same params structure like: 
```
KAFKA_CLUSTERS_0_ZOOKEEPER_PROPERTIES_CLIENTCNXNSOCKET
KAFKA_CLUSTERS_0_ZOOKEEPER_PROPERTIES_SECURE
...
```

**Review by germanosin — CHANGES_REQUESTED** [review]

File: docker/kafka-ui-zookeeper-ssl.yml (hunk 2 above)
Comment (Haarolean) [reply docker/kafka-ui-zookeeper-ssl.yml:27]: Done. And it works anyway, I dunno how it picks it up.

```14:41:35.603 [kafka-admin-client-thread | adminclient-1] DEBUG com.provectus.kafka.ui.service.ZookeeperService - Start getting Zookeeper metrics for kafkaCluster: local```

**Review by germanosin — APPROVED** [review]

**Pull request merged**